### DISCUSSION TITLE
UI font size

### DISCUSSION THREAD

**Discussion by Jucgshu** [jellyfin/jellyfin-desktop#329] [Q&A]
Hi guys,

Would it be possible to increase the UI font size? I can apply a zoom factor in the web UI, but I can't seem to find a way in JMP.

Thanks for your help :+1:

**Comment by JohnRo79** [comment]
this is the first post i've seen in regards to my issue too
but, in my case, i want to decrease it :)

<img width="822" height="604" alt="image" src="https://github.com/user-attachments/assets/ec11d1e4-568c-49b3-be01-dd53d3374342" />
